src/css/atoms/tooltip.css

```diff
@@ -0,0 +1,83 @@
+.tooltip {
+  background: $color-neutral;
+  border-radius: .25rem;
+  box-shadow: 0 0 3px 0 color($color-default-darker alpha(.3));
+  color: $color-default-dark;
+  font-size: .8rem;
+  opacity: 0;
+  padding: .5rem;
+  position: absolute;
+  transition: opacity .3s ease-in-out, visibility .6s;
+  visibility: hidden;
+
+  &:parent:hover > .tooltip {
+    opacity: 1;
+    visibility: visible;
+  }
+
+  .tooltip-arrow {
```

Comment:
I think this guy can move into an atom. And the positioning styles could go into the tooltip molecule